addons/project/models/project.py

```diff
@@ -1216,7 +1222,8 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         compute='_compute_has_late_and_unreached_milestone',
         search='_search_has_late_and_unreached_milestone',
     )
-
+    #used to display the stage of the task in the tree view (if the project is private will return the personnal_stage_id, else will return the stage_id)
+    stage_display = fields.Char(compute='_compute_stage_display', string='Stage')
```

Comment:
```suggestion
    stage_display = fields.Char(compute='_compute_stage_display', string='Stage Display')
```
The `stage_id` field already uses "Stage" as a label, and we shouldn't have two fields with the same label

Reply:
Done :)